airflow/migrations/versions/bbf4a7ad0465_remove_id_column_from_xcom.py

```diff
@@ -110,7 +110,8 @@ def upgrade():
             bop.drop_index('idx_xcom_dag_task_date')
             # mssql doesn't allow primary keys with nullable columns
             if conn.dialect.name != 'mssql':
-                bop.create_primary_key('pk_xcom', ['dag_id', 'task_id', 'key', 'execution_date'])
+                #bop.create_primary_key('pk_xcom', ['dag_id', 'task_id', 'key', 'execution_date'])
+                bop.create_primary_key('pk_xcom', ['dag_id', 'task_id', 'key'])
```

Comment:
Why this

Reply:
My mysql throws when creating the PK longer than 700. I have no idea how to fix it properly.

But since it's not related to utf8mb3，I will revert this line.